packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -617,6 +617,7 @@ const TRANSFORMS = {
       status: obj.status,
 
       $host: link(obj, 'resident_on'),
+      $ref: obj.$ref,
```

Comment:
I'm wondering if it's a good idea to expose the ref, we have not done this yet…

Reply:
I agree, I don't really like it either but I can't think of a better solution. Maybe object keys should be resolved to IDs when they're refs? So `vm.current_operations` would be a `{ <id>: <operation> }` map instead of `{ <ref>: <operation> }`. But I'm not sure about that either...

Reply:
I think that would be better yes.